client/src/components/FilterServicesAndFeatures/FilterServicesAndFeatures.tsx

```diff
@@ -60,7 +60,7 @@ export const FilterServicesAndFeatures = (props: Props) => {
   }
   // console.log('filteredServices onChange', filteredServices)
 
-  const handleResetFilter = (): any => {
+  const handleResetFilter = (): void => {
```

Comment:
Thanks for catching this one that I missed, @sergiobtos! Your efforts are appreciated!